service/topology_coordinator.cc

```diff
@@ -2719,6 +2719,8 @@ class topology_coordinator : public endpoint_lifecycle_subscriber {
                     node.guard = co_await exec_global_command(std::move(node.guard),raft_topology_cmd::command::barrier_and_drain, get_excluded_nodes(node), drop_guard_and_retake::yes);
```

Comment:
Please be more precise about affected places in the commit message. "raft:" should be used for changes in the Raft library (the `raft` directory). Something else should be used for changes in group 0. "group0" is fine, but it can also be something more specific like "topology_coordinator" in the case of this commit.

Also, "Raft commands" -> "topology commands".

I'm nitpicking, but I think we should try to be precise because soon the Raft-related stuff will become even more confusing with the strongly consistent tables.